Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1641,7 +1641,7 @@ class ExceptionHandlerInfo {
             }
         }
 
-        [TestMethod, Priority(0)]
+        [TestMethod, Priority(TestExtensions.CORE_UNIT_TEST)]
```

Comment:
based on the category 60s, looks like `TestExceptions` may be a slow test, if it is slow, you could demote it.

Reply:
Changed